swagger/swagger.yaml

```diff
@@ -593,6 +612,36 @@ components:
       description: >
         This object describes Redis container.
 
+    memcached-request:
+      type: object
+      properties:
+        preset:
+          $ref: '#/components/schemas/memcached'
+        options:
+          $ref: '#/components/schemas/options'
+      description: >
+        This request includes Memcached and general configuration.
+
+    memcached:
+      type: object
+      properties:
+        values:
+          type: object
+          description: A list of key/value pairs to create in the container. Values are base64 strings.
+          additionalProperties:
+            oneOf:
+              - type: string
+          example:
+            foo: YmFy
+            baz: NDI=
+            meh: My4xNA==
```

Comment:
Values/Byte values need to be supported here as well.

Reply:
Done. I've also added in [`24f7777` (#47)](https://github.com/orlangure/gnomock/pull/47/commits/24f777762a28288a05dae7bfba1c01e62f38521c) a string format. Swagger supports a specific [string format](https://swagger.io/docs/specification/data-models/data-types/) that suggests to the user what should data look like. This fits perfectly into the idea of ByteValues.